lvgl/src/widgets/label.rs

```diff
@@ -10,6 +13,16 @@ impl Label {
     }
 }
 
+#[cfg(feature = "alloc")]
+impl<S: AsRef<str>> From<S> for Label {
+    fn from(text: S) -> Self {
+        let text_cstr = CString::new(text.as_ref()).unwrap();
```

Comment:
This is a test of what is possible with `alloc` feature enabled. Actualy the `From` trait implementation should use the `TryFrom` trait implementation and just call `.unwrap()` here. People will decide which guarantees they want to have.